spec/requests/api/v1/articles_spec.rb

```diff
@@ -13,8 +14,98 @@
     allow(FeatureFlag).to receive(:enabled?).with(:api_v1).and_return(true)
   end
 
-  describe "GET /api/articles" do
-    before { article }
+  describe "GET /articles" do
+    before do
+      article.update_columns(organization_id: organization.id)
+    end
+
+    # rubocop:disable RSpec/VariableName
+    path "/api/articles" do
+      get "Published articles" do
+        tags "articles"
```

Comment:
Many of these keywords are specific to generated OpenAPI/Swagger docs. The existing docs for V0 of the API are viewable at https://github.com/forem/forem-docs/blob/main/api_v0.yml